cvat-ui/src/components/quality-control/quality-settings-tab.tsx

```diff
@@ -48,14 +100,74 @@ function QualitySettingsTab(props: Readonly<Props>): JSX.Element | null {
         );
     }
 
+    let header: JSX.Element | null = null;
+    if (instance instanceof Task && instance.projectId !== null) {
+        header = (
+            <div className='cvat-quality-control-settings-header'>
+                <Switch checked={settings?.inherit} onChange={onInheritChange} />
```

Comment:
`settings` may be null in case of fetching error as I understand
But should we render anything in this case if rendered elements will not work? 

E.g. if there is no settings, this switcher won't do anything. And probably other uncertain issues appear. So, maybe just not to render the component?

Reply:
There is a conditional rendering that will render simple text if there is no settings [below](https://github.com/cvat-ai/cvat/blob/develop/cvat-ui/src/components/quality-control/quality-settings-tab.tsx#L53)

Reply:
So, if request has failed, why "No quality settings found"? That does not seem like truth message

Reply:
Removed. Now component returns `null`